recipes/desman/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+$PYTHON setup.py install
```

Comment:
Try to set it to `$PYTHON setup.py install --single-version-externally-managed --record=record.txt`